src/bindings/c/src/ov_partial_shape.cpp

```diff
@@ -5,62 +5,80 @@
 
 #include "common.h"
 
-ov_status_e ov_partial_shape_create(ov_partial_shape_t** partial_shape_obj, ov_rank_t* rank, ov_dimensions_t* dims) {
-    if (!partial_shape_obj || !rank) {
+bool check_dimension(const ov_dimension_t* dims, int64_t size) {
+    for (auto i = 0; i < size; i++) {
+        auto& _dim = dims[i];
+        if (_dim.max < -1 || _dim.min < -1 || _dim.max < _dim.min)
```

Comment:
What if dimension is -1? What does it mean?

Reply:
-1 means dynamic.